core/ui/public/i18n/en/translation.json

```diff
@@ -544,10 +544,10 @@
     "samba_placeholder":"Share name",
     "smb_short": "SMB",
     "samba_tooltips":"Only shares with SMB2/3 protocols are supported. Deprecated SMB1/CIFS shares are not secure and cannot be accessed.",
-    "cluster": "Local storage",
```

Comment:
This i18n string is used in the first step of the wizard (repo type selection), let's not remove it

Reply:
sorry it was a duplicate json property, I should have explained it : https://github.com/NethServer/ns8-core/blob/fixSmbValidation/core/ui/public/i18n/en/translation.json#L394